app.js

```diff
@@ -10,6 +10,14 @@ const repository = require('./repository');
 const util = require('./util');
 const originalQuery = mu.query;
 
+const SERVICENAME = 'agenda-approve-service';
+const GRAPH = 'http://mu.semte.ch/graphs/public';
+
+const errorLoggingModule = require('error-logging-module');
+
+const logger = new errorLoggingModule();
+logger.setGraph(GRAPH);
+logger.setServiceName(SERVICENAME);
```

Comment:
GRAPH is better named PUBLIC_GRAPH to avoid confusion with other graphs

Reply:
solved